pyproject.toml

```diff
@@ -0,0 +1,112 @@
+[project]
+name = "gaussianfft"
+dynamic = ["version"]
+license = { file = "LICENSE.txt" }
+description = "A fast library for simulating Gaussian Random Fields, using the fast Fourier transform"
+readme = "README.md"
+authors = [
+    { name = "Norwegian Computing Center", email = "" },
+    { name = "Sindre Nistad", email = "snis@equinor.com" },
+]
+keywords = [
+    "Stochastic simulation",
+    "Gaussian random field",
+    "FFT",
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: C++",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dependencies = [
+    "numpy",
+]
+requires-python = ">=3.6"
+
+[project.urls]
+Homepage = "https://equinor.com"
+Repository = "https://github.com/equinor/gaussianfft"
+Issues = "https://github.com/equinor/gaussianfft/issues"
+
+[project.optional-dependencies]
+util = ["scipy"]
+
+
+[build-system]
+requires = [
+    # Scikit-build-core does not support Python 3.6 (and older)
+    "scikit-build-core ; python_version >= '3.7'",
+
+    "mkl-static ; platform_machine == 'x86_64' or platform_machine == 'AMD64'",
+    "mkl-devel ; platform_machine == 'x86_64' or platform_machine == 'AMD64'",
+
+    # These are the versions of NumPy used by RMS
+    "numpy==1.24.3 ; python_version == '3.11'",  # 14.2.0 and newer
+    "numpy==1.19.2 ; python_version == '3.8'",  # 12.1.0 and newer
+    "numpy==1.13.3 ; python_version == '3.6'",  # 11.0.0 and newer
```

Comment:
Can we remove this line?

Reply:
In theory, but it is still possible to compile it for Python 3.6, so I figured it should be there for the time being.

On a side-note, I'm also working on using pybind11 instead of Boost.Python, which does not depend on lumpy being installed during the build step